### PR TITLE
add chmod +x cmd to deweb local server docs

### PR DESCRIPTION
**Pull request by pivilartisant**

### CODE REVIEW

File: docs/deweb/getting-started/setup-local-server.mdx [hunk 1]
```diff
@@ -52,7 +52,13 @@ For Linux users, running binaries via GUI can vary greatly depending on the desk
 
 1. Open your terminal (command prompt on Windows).
 2. Navigate to the directory where the binary is located.
-3. Run the server with the following command:
+3. Run the following command to allow execution permission:
+
+```bash
+   chmod +x deweb-server
+```
+
```
Comment (thomas-senechal) [inline docs/deweb/getting-started/setup-local-server.mdx]: That's only for Macos and linux

File: docs/deweb/getting-started/setup-local-server.mdx [hunk 2]
```diff
@@ -52,7 +52,13 @@ For Linux users, running binaries via GUI can vary greatly depending on the desk
 
 1. Open your terminal (command prompt on Windows).
 2. Navigate to the directory where the binary is located.
-3. Run the server with the following command:
+3. Run the following command to allow execution:
+
+```bash
+   chmod +x deweb-server
+```
```
Comment (thomas-senechal) [inline docs/deweb/getting-started/setup-local-server.mdx:59]: Github ui is broken so I can't do the suggestion, but you need to add indentation to make it look better in the ui

File: docs/deweb/getting-started/setup-local-server.mdx [hunk 3]
```diff
@@ -72,3 +79,7 @@ To customize your server settings, refer to the [Server Configuration](../config
 DeWeb is still a work in progress, and we are actively listening to feedback.
 If you encounter any issues or have suggestions, please let us know through our [issue tracker](https://github.com/massalabs/DeWeb/issues).
 :::
+
+```
+
+```
```
Comment (thomas-senechal) [inline docs/deweb/getting-started/setup-local-server.mdx:85]: What is that ? 🤔

Comment (pivilartisant) [reply docs/deweb/getting-started/setup-local-server.mdx]: no idea i remove them

**Review by damip — APPROVED** [review]

**Pull request merged**